.github/ISSUE_TEMPLATE/new_repo_request.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Is the plan to have new repo request issues be opened against the `.github` repo?

Reply:
the ```.github``` repo seems like a logical place to open these requests, since the contributing guide is also maintained here

we could also have a separate repo for them but that seems to be overkill?